lib/fog/kubevirt/compute/compute.rb

```diff
@@ -523,6 +523,10 @@ class Mock
         def initialize(options={})
         end
 
+        def namespace
+          "default"
```

Comment:
this cannot be static value, it should be taken form the argument provided by the user by `kubevirt_namespace` and stored in `@namespace`

Reply:
In principal I agree that we should accept `namespace` and create instance variable. On the other hand who it would change our mocks? We have bunch of hard coded responses and we need this field any to be able to call `vms.create`.
How changing value would affect what mock mode is returning?